src/carousel/carousel.service.ts

```diff
@@ -29,6 +31,8 @@ export class CarouselService {
     createCarouselSlideFilesDto: CreateCarouselSlideFilesDto,
   ) {
     const { image } = createCarouselSlideFilesDto;
+    const { title, lang, subtitle } = createCarouselSlideDto;
+    const language = lang ?? Language.En;
```

Comment:
Similar thing as before. You can set a default value in a single line like so: `const { title, lang = Language.English, subtitle } = createCarouselSlideDto;`

Also, lets stick with either `lang` or `language` for consistency. I know that frontend prefers lang for shorter params in URLs but I'm all up for clarification and full words, so `language` is my preference. Your call